docs/docs/directives/schema-directives.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_position: 1
+---
+
+# Schema Directives
+
+Schema Directives are used to decorate the schema to provide more information to clients, EntityGraphQL can also use this information to update the introspection model or provide extra functionality such as validation.
+
+## Built-In Directives
+
+The GraphQL spec defines directives that are supported out of the box in EntityGraphQL.
+
+- `@deprecated(reason: String)` - Tells the client that this type, field or enum value should no longer be used along with the reason why or a suggested alternative
+- `@oneOf` - Skip this field if the argument is true.
```

Comment:
Copy and paste error with the description of this one and the next.

Can we also link to the `oneOf` proposed spec (https://github.com/graphql/graphql-spec/pull/825)